configuration/src/main/java/io/airlift/configuration/ConfigBinder.java

```diff
@@ -110,6 +110,15 @@ private ConfigBinder(Binder binder)
         createConfigDefaultsBinder(key).addBinding().toInstance(new ConfigDefaultsHolder<>(key, configDefaults));
     }
 
+    /**
+     * Binds default values for all the instances of given config class for the current binder
+     */
+    public <T> void bindConfigGeneralDefaults(Class<T> configClass, ConfigDefaults<T> configDefaults)
```

Comment:
`bindConfigGeneralDefaults` binds defaults for all the instances of that particular configuration class. e.g. `key(ConfigClass.class, Annotation1.class)`, `key(ConfigClass.class, Annotation2.class)`,  `key(ConfigClass.class)`

`bindConfigDefaults` binds defaults for the instance with only the `Class` in key. e.g.: `key(ConfigClass.class)`